malaria24/ona/migrations/0025_smsevent.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ona', '0024_inboundsms'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SMSEvent',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('event_type', models.CharField(max_length=255)),
+                ('timestamp', models.DateTimeField()),
+                ('sms', models.ForeignKey(to='ona.SMS')),
+            ],
+        ),
+    ]
```

Comment:
I think this is the wrong migration? This is for `SMSEvent`, not `InboundSMS`, and it references a non-existing `0024_inboundsms` migration.

Reply:
:woman_facepalming: